.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like we did not include this before which I don't think will hurt anything but after reading [this](https://stackoverflow.com/questions/65754188/should-i-add-swiftpm-to-gitignore) we may want to include it.

Reply:
Not so sure we should add this. All of our testing has been without it and things have worked so far. So why change. I read the post, and it didn't convince me. But perhaps get a second opinion for @philium